website/docs/help-and-support/troubleshooting-guide/query-errors.md

```diff
@@ -5,6 +5,16 @@ description: >-
 # Query Errors
 The section illustrates common query errors and how to resolve them on Appsmith.
 
+## Find error messages from the datasource
+
+When a query fails, Appsmith commonly shows a notification stating something like: "UncaughtPromiseRejection: <query_name> failed to execute."
```

Comment:
should we add "UncaughtPromiseRejection: <query_name> failed to execute." inside code blocks?